### PR TITLE
Task name hyphenation + ALCA/ALCB in task names

### PR DESCRIPTION
**Pull request by keirongulrajani**
## Scope

- Updated task names to use hyphenated standard
- Replaced some instances of AToken/BToken with ALCA/ALCB

## Why?

[MP-858](https://madhive.atlassian.net/browse/MP-858)
[MP-858](https://madhive.atlassian.net/browse/MP-859)

## Todos

If any, what are the follow-up tasks required other than merging this PR? Have they been arranged?

- [ ] ???

### CODE REVIEW

File: bridge/scripts/lib/alicenetFactoryTasks.ts [hunk 1]
```diff
@@ -286,8 +286,8 @@ task("deployContracts", "runs the initial deployment of all AliceNet contracts")
   });
 
 task(
-  "fullMultiCallDeployProxy",
-  "Multicalls deployCreate, deployProxy, and upgradeProxy, if gas cost exceeds 10 million deployUpgradeableProxy will be used"
+  "full-multi-call-deploy-proxy",
```
[the remark this answers is not in the document]
Comment (keirongulrajani) [reply bridge/scripts/lib/alicenetFactoryTasks.ts:289]: Good catch! looks like it does call that, I'll fix it

File: bridge/README.md [hunk 2]
```diff
@@ -69,17 +69,17 @@ If you are linking a custom Deployment list file it must follow the following fo
...
 --output-name <filname> <contract1> <contract2>
-change all npx hardhat DeployContracts --list-path path --arg-file-path path single proxy deployment npx hardhat
-deployUpgradeableProxy --contract-name <name>
-single metamorphic deployment npx hardhat deployMetamorphic --contract-name <name>
-incremental updates upgrade proxy logic non multiCall npx hardhat deployCreate --contract-name logicContractName npx
-hardhat upgradeDeployedProxy --contract-name <name> --logic-address  
-multiCall npx hardhat multiCallUpgradeProxy --contractName <name>
+change all npx hardhat deploy-contracts --list-path path --arg-file-path path single proxy deployment npx hardhat
+deploy-upgradeable-proxy --contract-name <name>
+single metamorphic deployment npx hardhat deploy-metamorphic --contract-name <name>
```
Comment (z-j-lin) [inline bridge/README.md:79]: this is no exists

**Review by z-j-lin — CHANGES_REQUESTED** [review]
remove deploymetamorphic from readme

**Review by z-j-lin — APPROVED** [review]
looks good

**Review by vtleonardo — APPROVED** [review]

**Pull request merged**